ponyo/vae.py

```diff
@@ -221,10 +223,9 @@ def tybalt_2layer_model(
     intermediate_dim,
     latent_dim,
     epsilon_std,
-    rnaseq,
```

Comment:
i don't remember this config param?

Reply:
This is the normalized compendium expression data that is found in the "normalized_compendium_filename" param